cts/lab/CTStests.py

```diff
@@ -1222,7 +1222,7 @@ def errorstoignore(self):
         '''Return list of errors which should be ignored'''
         return [
             r"Updating failcount for %s" % self.rid,
-            r"schedulerd.*: Recover %s\s*\(.*\)" % self.rid,
+            r"schedulerd.*: Recover\s*%s\s*\(.*\)" % self.rid,
```

Comment:
This is zero or more spaces (rather than one or more spaces), if that matters.

Reply:
Whoops, updated commit

Reply:
There are a bunch more instances of `\s*` in `patterns.py`, but the updated commit hits all the ones for `Recover` messages as desired.